web-client/src/components/NewRequest/NewRequest.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, Descriptions, Form, Input } from "antd";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+
+import TitleWithAddon from "../TitleWithAddon/TitleWithAddon";
+
+const MainDiv = styled.div`
+  display: flex;
+`;
+
+const IconDiv = styled.div`
+  width: 25%;
+  padding-right: 24px;
+`;
+
+const IconImage = styled.img`
+  width: 100%;
+`;
+
+const FormDiv = styled.div`
+  width: 75%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+`;
+
+interface NewRequestProps {
+  handleFormSubmit: Function;
+}
+
+const NewRequest: React.FC<NewRequestProps> = ({
+  handleFormSubmit
+}): React.ReactElement => {
+  const { t } = useTranslation();
+  const [form] = Form.useForm();
+
+  return (
+    <MainDiv>
+      <IconDiv>
+        {/* ATENTION!! THIS IS WRONG, JUST A HACK UNTIL I FIND SOMETHING BETTER */}
+        <IconImage src="https://reach4help.org/static/logo-78d2063bcc5e542bf67195e3c6fb6fcf.svg" />
```

Comment:
You can create a JS file with the SVG and import the SVG here

Reply:
I think you can also import the SVG file directly

Reply:
Fixed here https://github.com/reach4help/reach4help/pull/247/commits/ca319d8e0c54f28d0dd58589728db16682704ab7

Reply:
You can delete the comment since it doesn't have effect now

Reply:
Done here - https://github.com/reach4help/reach4help/pull/247/commits/da0ef6507096e1b0b836ff774cea8eb490190680